transform/app/services/solr/validate.py

```diff
@@ -35,29 +42,27 @@ def check_document_exists(
                 if response_data["response"]["numFound"] > 0:
                     check_if_exists.append(True)
                     logger.info(
-                        f"Document found. Data type={col_name}, solr_col={s_col_name}, ID={data_id}"
+                        f"Document ID={data_id} found in solr_col={s_col_name} (data type: {col_name})."
                     )
                 else:
                     check_if_exists.append(False)
                     logger.info(
-                        f"No document found with ID. Data type={col_name}, solr_col={s_col_name}, ID={data_id}"
+                        f"No document ID={data_id} found in solr_col={s_col_name} (data type: {col_name})."
                     )
             else:
                 check_if_exists.append(False)
                 logger.error(
-                    f"{req.status_code} Failed to query Solr."
-                    f"Data type={col_name}, solr_col={s_col_name}, ID={data_id}"
+                    f"Failed to query Solr ({url=}). Status: {req.status_code}."
+                    f"Data type: {col_name}, Solr collection: {s_col_name}, ID: {data_id}"
                 )
-
         except ReqConnectionError as e:
             check_if_exists.append(False)
             logger.error(
-                f"Connection failed {url=}."
-                f"Querying Solr has failed. Data type={col_name}, solr_col={s_col_name}. Details: {e}"
+                f"Connection failed to {url=}."
```

Comment:
Solr connection failed, {url=}. Data type={col_name}, solr_col={s_col_name}. Details: {e}

Reply:
You are right, it makes it easier to read.